week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,6 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
+  return firstWord + secondWord + thirdWord;
 }
 
-var result = concatenate('code', 'your', 'future');
-console.log(result); // expected result = "codeyourfuture"
\ No newline at end of file
+var result = concatenate("code", "your", "future");
+console.log(result); // expected result = "codeyourfuture"
```

Comment:
Have a look at MDN ( https://developer.mozilla.org/en-US/) and see if you can find a JavaScript 'String' method for achieving the same thing.